CHARTER.md

```diff
@@ -0,0 +1,71 @@
+**End Date:** 31 May 2021 
+
+**Confidentiality:** Proceedings are [public](https://www.w3.org/2005/10/Process-20051014/comm.html#confidentiality-levels) 
+
+**Chairs:** Johannes Wilm (Invited Expert), Grisha Lyukshin (Microsoft)
```

Comment:
Chairs doesn't seem right... this isn't a working group. 
```suggestion
**Chairs:** Johannes Wilm (Invited Expert), Grisha Lyukshin (Microsoft)
```

Reply:
@marcoscaceres Is that following W3C procedures? We had been informed that taskforces can have their own chairs. Were we incorrectly informed about that?

Reply:
As per the process document - task forces are just informal things: 

> A Chair may form task forces (composed of group participants) to carry out assignments for the group. The scope of these assignments must not exceed the scope of the group's charter. A group should document the process it uses to create task forces (e.g., each task force might have an informal "charter"). Task forces do not publish technical reports; the Working Group may choose to publish their results as part of a technical report.

We are already kinda violating the process document by allowing the task force to publish technical reports... though we will publish anything the task force produces as part of the Working Group. 

The thing is not to cause confusion - the task force is not a Working Group.

Reply:
The formal W3C Process has very little to say about Task Forces. See the last paragraph of https://www.w3.org/2019/Process-20190301/#ReqsAllGroups.

Based on that "Chair of a task force" is not a defined official term. Neither is "Coordinator of a task force". I'd say using one word or the other is a subjective matter of taste. Using the word chair is familiar, and people will know what it means. But it may also lead to some confusions, as people might assume that the role has some formal powers that it doesn't actually have (other than by delegation from the actual WG's chairs).

Reply:
> But it may also lead to some confusions, as people might assume that the role has some formal powers that it doesn't actually have (other than by delegation from the actual WG's chairs).

Yeah, that's what I'm worried about too. Thanks @frivoal.

Reply:
I think other WGs have Task Forces which call their coordinators "chairs", (I believe that's the case for Houdini for instance), so there's precedent. Maybe a middle ground to limit confusion while avoiding an unfamiliar term would be to be explicit in the TF's charter that the TF chairs are acting as delegates from the WG chairs, and that in case of doubt or conflict, the WG chairs have the final word (and final responsibility).

Reply:
Sounds good. I can live with that.

Reply:
I am fine with both approaches but I like the alternative @frivoal is proposing.
Should we then create a "definitions" section where we define what chairs mean in this charter?

Reply:
I'd just include it inline.

Reply:
done.